exporter/tanzuobservabilityexporter/README.md

```diff
@@ -46,14 +53,25 @@ exporters:
 service:
   pipelines:
     traces:
-      receivers: [examplereceiver]
-      processors: [batch]
-      exporters: [tanzuobservability]
+      receivers: [ examplereceiver ]
+      processors: [ batch ]
+      exporters: [ tanzuobservability ]
 ```
 
 ### Advanced Configuration
 
-This exporter uses [queuing and retry helpers](https://github.com/open-telemetry/opentelemetry-collector/blob/main/exporter/exporterhelper/README.md) provided by the core OpenTelemetry Collector. The `retry_on_failure` and `sending_queue` features are enabled by default, but can be disabled using the options below.
+#### Processor
+
+The memory limiter processor is used to prevent out of memory situations on the collector. It allows performing periodic
+checks of memory usage if it exceeds defined limits will begin dropping data and forcing GC to reduce memory
```

Comment:
I think we need to clean up some grammar. Something like, "It allows performing periodic
checks of memory usage – if it exceeds defined limits it will begin dropping data and forcing garbage collection to reduce memory consumption."

Reply:
Will do.